golang/main.go

```
@@ -40,19 +41,31 @@ func homePage(w http.ResponseWriter, r *http.Request) {
-		log.Println("Error in bridgeToken\n", err)
-		fmt.Fprintf(w, `{ "success": false }`)
+	isOrder := os.Getenv("IS_ORDER")
+	
+	if strings.ToLower(isOrder) == "true" {
+		orderData, err := createOrder()
+		if err != nil {
+			log.Println("Error creating order", err)
+			fmt.Fprintf(w, `{ "success": false }`)
+			return
+		}
+		fmt.Fprintf(w, orderData)
```

golangci-lint (1.64.8)

SA1006: printf-style function with dynamic format string and no further arguments should use print-style function instead

(staticcheck)